bfvmm/src/hve/arch/intel_x64/delegator/cpuid.cpp

```diff
@@ -122,21 +122,17 @@ delegator::add_handler(cpuid::leaf_t leaf, const cpuid::delegate_t &d)
 bool
 delegator::handle(vcpu_t vcpu)
 {
-    const auto &hdlrs = m_handlers.find(vcpu->rax());
-    struct info_t info = { 0, 0, 0, 0, false, false };
+    auto leaf = vcpu->rax();
+    auto subleaf = vcpu->rax();
```

Comment:
Dangit... copy paste error. I'll force push this out in one minute, should be ```auto subleaf = vcpu->rcx();```

Reply:
Actually, I may have messed up the terminology "leaf" vs "subleaf". I was thinking that leaf = eax and subleaf = ecx. Is that wrong?